tests/unit/components/grapher/Dot.spec.ts

```diff
@@ -0,0 +1 @@
+// Unsure on what to do here...
```

Comment:
I tried a bunch trying to get the "Dot" mounted, but I couldn't figure it out... Any help here?

Reply:
What's the error you're seeing?